azure-pipelines.yml

```diff
@@ -54,6 +54,11 @@ stages:
         python setup.py sdist bdist_wheel
       workingDirectory: $(baseFolder) 
       displayName: 'Building'
+    - task: ManifestGeneratorTask@0
+      displayName: "SBOM Generation Task"
+      inputs:
+          BuildDropPath: "$(Build.ArtifactStagingDirectory)"
```

Comment:
I think this should be 'dist'? ManifestGeneratorTask and PublishBuildArtifacts need to be using the same path

Reply:
Good catch, no wonder it wasn't working before :)